### PR TITLE
feat: cms content

### PR DESCRIPTION
**Pull request by supproduction**
Moves cms pages from labs

closes: [CC-34208](https://spryker.atlassian.net/browse/CC-34208)

### CODE REVIEW

File: libs/template/presets/storefront/experience/pages/cms-pages.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { cmsTypes } from '@oryx-frontend/content';
+import { ExperienceComponent } from '@oryx-frontend/experience';
+
+export const cmsPages: ExperienceComponent[] = cmsTypes.map((type) => ({
```
Comment (tolerants) [inline libs/template/presets/storefront/experience/pages/cms-pages.ts]:
```suggestion
export const cmsPage: ExperienceComponent[] = cmsTypes.map((type) => ({
```

**Review by tolerants — CHANGES_REQUESTED** [review]

File: libs/template/presets/storefront/experience/pages/cms-pages.ts (hunk 1 above)
Comment (supproduction) [reply libs/template/presets/storefront/experience/pages/cms-pages.ts]: But we define here multiple pages

Comment (tolerants) [reply libs/template/presets/storefront/experience/pages/cms-pages.ts]: As for me we are defining a common typical cms page here, not a few specific ones.

Comment (supproduction) [reply libs/template/presets/storefront/experience/pages/cms-pages.ts]: Fixed

Comment (tolerants) [reply libs/template/presets/storefront/experience/pages/cms-pages.ts]: Can you please also fix a file name?

**Review by tolerants — APPROVED** [review]

**Pull request merged**